docs/source/firehose.rst

```diff
@@ -15,24 +15,26 @@ Configuration
 
 When enabling the Kinesis Firehose module, a dedicated Delivery Stream is created per each log type.
 
-For example, if the ``sources.json`` defines the following:
+For example, if the data_sources field for a cluster defined in ``conf/clusters/<cluster>.json`` looks like the following:
```

Comment:
Please re-read this sentence. I'm not sure what you're going for here exactly. this also sorta implies that `data_sources` is optional within the cluster's config